test/manipulation.test.js

```diff
@@ -743,7 +743,15 @@ describe('manipulation', function() {
       Todo = db.define('Todo', {
         content: String,
       });
-      db.automigrate(['Post', 'Todo'], done);
+      Person = db.define('Person', {
+        name: String,
+        gender: String,
+        married: Boolean,
+        age: {type: Number, index: true},
+        dob: Date,
+        createdAt: {type: Date, default: Date},
```

Comment:
@bajtos It is created to override the `Person` defined outside the describe('updateOrCreate'), because [test case in updateOrCreate](https://github.com/strongloop/loopback-datasource-juggler/blob/67fd2121e3a32521f18d6db2939860199121b9b8/test/manipulation.test.js#L833) needs a `Person` model with `forceId: false`